### DISCUSSION TITLE
Is There a Method for Adding Upscalers?

### DISCUSSION THREAD

**Discussion by bitshifter52** [Coyote-A/ultimate-upscale-for-automatic1111#34] [Q&A]
Pardon me if I'm not asking this correctly - I'm new to A1111 and your upscalers. Is there a way to add additional upscale files to your addon?

There are Remacri, Valar, UltraSharp which are mentioned in this post https://www.reddit.com/r/StableDiffusion/comments/xsoczh/upscalers_comparison_part_2_swinir_remacri_valar/ and I am hoping to give them a try.

Anyway, thank you for your upscaler system, it's very helpful.

**Comment by WalternateBishop** [comment]
Just download them from [https://upscale.wiki/wiki/Model_Database](https://upscale.wiki/wiki/Model_Database) and put into your Automatic1111's \models\ESRGAN folder, unless they're SwinIR models, in that case put them in the SwinIR folder.
